examples/README.md

```diff
@@ -0,0 +1,159 @@
+Creating a basic S2I builder image  
+---
+
+### Getting started  
+
+Directory skeleton:  
+
+* `Dockerfile` – standard Dockerfile where we’ll define the base builder image.  
+* `assemble` - responsible for building the application.  
+* `run` - responsible for running the application.  
+* `usage` - optional script responsible for printing the usage of the builder.  
+* `save-artifacts` - optional script for incremental builds that save built artifacts.  
+
+We are going to start by adding our logic to the `Dockerfile`.  
```

Comment:
maintaining this content in two places (here and in the actual file) is going to be a problem.  just link to it.

same for the other scripts.